README.md

```diff
@@ -25,7 +25,7 @@ A library and CLI app for rendering project templates.
 
 1. Install Python 3.9 or newer.
 1. Install Git 2.27 or newer.
-1. To use as a CLI app: `pipx install copier`
+1. To use as a CLI app: [`pipx install copier`](https://github.com/pypa/pipx) or [`uv install copier`](https://docs.astral.sh/uv/#tool-management)
```

Comment:
I believe it should be `uv tool install copier`?

Reply:
yes, sorry -_-

changed